x-pack/legacy/plugins/watcher/common/models/action/email_action.js

```diff
@@ -108,11 +108,9 @@ export class EmailAction extends BaseAction {
         code: ERROR_CODES.ERR_PROP_MISSING,
         message
       });
-
-      json.email = {};
     }
 
-    if (!json.email.to) {
+    if (json.email && !json.email.to) {
```

Comment:
@sebelga Would you mind reviewing these types of validation changes I've made in the various action files? I believe you worked on some of this code and I want to make sure this logic still behaves as intended.